### PR TITLE
Fixes #319 - Time/date of alert - center text

### LINKED ISSUE #319: Width is missing - shrinked content in popup/alert page + alignment

### PR DESCRIPTION
**Pull request by Eldenroot**
Add width tag and add align missing tag

Fixes #319

### CODE REVIEW

**Comment by Eldenroot** [comment]
Width is not added for these templates, so if the thread has a long title -> it shrinks the column for date/time. Please try it on your own. Also I added "center" for date/time alignment. Anyway I can update the PR and delete the width change.

**Comment by lairdshaw** [comment]
> Please try it on your own.

I've just done that. I can't reproduce a shrunken date/time column. When the thread title is overlong, the popup stretches to fit, although the close button "X" doesn't shift along with it. I think we can live with that minor glitch. See the image below for what I'm seeing.

Screenshot 2024-02-15 at 19-57-02 Recent Alerts - Forums (1)

Maybe you have added custom styling somewhere like a `max-width` which is causing this problem on your own board? Or maybe I've forgotten about something I've done on my vanilla dev board which is fixing the problem.

**Comment by Eldenroot** [comment]
It is weird... maybe because of Czech language MyBB core pack?

**Comment by lairdshaw** [comment]
Oh, wait, I see that with a longer date ("Less than 1 minute ago")  it actually does appear shrunken. Is this the sort of thing you mean?

Screenshot 2024-03-07 at 13-09-47 0123456789012345678901234567890123456789012345678901234567890123456789012345678901234

**Comment by Eldenroot** [comment]
Yes

**Comment by lairdshaw** [comment]
Hmm. It seems that that's an extreme scenario, with no breaking whitespace in the thread's subject. When I insert regular whitespace characters, it seems fine:

Screenshot 2024-03-07 at 15-47-30 Recent Alerts - Forums (1)

And when I remove a few "words" from the title, it also looks fine:

Screenshot 2024-03-07 at 15-46-49 Recent Alerts - Forums (1)

Do you have any examples other than extreme scenarios like that of my previous post that you can share?

**Comment by Eldenroot** [comment]
Almost any longer thread name

**Comment by lairdshaw** [comment]
Screenshot?

**Comment by lairdshaw** [comment]
@Eldenroot, sorry, but I don't think that the issue this PR solves is really an issue (aside from the alignment fix). I think browsers do a satisfactory job of managing column widths without explicit width styling, and I also think that when an explicit width is provided, oftentimes it is unnecessarily long and thus over-truncates the *alert description* column, causing another problem of the type it was designed to fix.

To ease your disappointment, please note that PR #323 increases the overall width of the modal popup to make room for the new "Unread only" checkbox.

I'd be happy to merge this PR if you remove the width styling and simply keep the alignment styling.

Alternatively, maybe @euantorano disagrees with me and agrees with you, in which case, the PR can be merged as-is.

**Comment by Eldenroot** [comment]
@lairdshaw - reverted changes, please merge

File: inc/plugins/MybbStuff/MyAlerts/templates/alert_row.html
Comment (lairdshaw) [inline inc/plugins/MybbStuff/MyAlerts/templates/alert_row.html:3]: As best I can tell, the `width` attribute takes only a number or a number followed by a percentage sign, not a number followed by "px". I might be wrong about this, but if I'm not, this change should be reverted before we merge the PR.

**Review by lairdshaw — CHANGES_REQUESTED** [review]

**Comment by Eldenroot** [comment]
Yes, you are righz. I missed this.

**Comment by lairdshaw** [comment]
Not to worry; I made the change for you.

**Pull request merged**
